docs/howto/types.md

```diff
@@ -0,0 +1,65 @@
+(types)=
+# How to prepare your Go types 
+
+To use your Go types with SQLair you will need to do some preparation. SQLair
+supports inputs from structs maps and slices and can output into structs
+and maps. 
+
+In general, all types used with SQLair need to be named, otherwise there is no
+way to reference them in the query. The rest of the preparation of the type
+depends on what kind it is.
+
+## Structs
+
+To use a struct with SQLair you must tag all the fields that correspond to a
```

Comment:
This "must" -- is it trying to describe a prerequisite or rather a step? Are there other steps? If not, just say:

> To prepare a struct Go type for use with SQLair, tag all the fields that correspond...

Basically, "must" is describing a truth, but here you're trying to give steps, so adjust the perspective accordingly. PS Also relevant for further content below.